andela/travel_tool_front
# #161583635 User profile page mobile responsiveness

#### What does this PR do?
This PR makes the `User Profile` page responsive.

#### Description of Task to be completed?
- Update the `user profile page` stylesheet.

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Enter into the project directory - `cd travel_tool_front`
-  checkout to this branch `git checkout ch-profile-page-responsiveness-161583635`
- Install dependencies and start the  app - `make start`
-  [Start](https://github.com/andela/travel_tool_back/pull/139)  the backend.
- Navigate to [travela](http://travela-local.andela.com:3000) in your browser and login into the app. 
- Click on the `Settings` tab and select `User Profile`
- Open the browser console and click on the `toggle device toolbar` button
- From the `device toolbar` choose different viewports
- The page should be responsive to different viewports

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#161583635](https://www.pivotaltracker.com/story/show/161583635)

#### Screenshots (if appropriate)
![responsiveness](https://user-images.githubusercontent.com/27569442/47850682-5e526c80-ddd6-11e8-8e93-24e51d35b203.gif)


#### Questions:
N/A
